README.md

```diff
@@ -4,6 +4,17 @@ This repository contains unofficial patterns, sample code, or tools to help deve
 
 # Fauna Schema Migrate
 
+## Deprecation Notice
+
+**This project is deprecated and no longer in development** 
+
+Fauna has introduced a suite of officially supported schema languages and tooling which fully replace Fauna Schema Migrate. These are the [Fauna Schema Language](https://docs.fauna.com/fauna/current/cookbook/advanced/fsl/get_started_fsl) (FSL),  [Fauna Command Line Interface](https://docs.fauna.com/fauna/current/tools/shell/) (Fauna CLI), and [GitHub and GitLab integrations](https://fauna.com/blog/supercharge-your-devops-workflow-with-fauna-schema-language). Combined, these features allow you to apply DevOps priciples to database development, and incorporate schema-level changes into broader CI/CD workflows.
+
+**The Fauna Schema Language** is a declarative language for expressing your entire database schema as a set of .fsl files. FSL files are intended to be managed in the same fashion as users manage their application code. That is, FSL files can be managed in common version revision control systems, where they can be shared, peer reviewed, version tracked, and where changes are automated into users' continuous integration and continuous deployment pipelines.
+
+**The Fauna Command Line Interface** has been expanded to support management of database schema. The Fauna CLI is a standalone tool through which users use FQL to send queries, upload CSV files, and now most recently, upload or download FSL files to and from your database. It is designed to be integrated into your development workflows, and is therefore the recommended way to manage schema.
```

Comment:
- "database schema management" reads better than "management of database schema".
- Some comma redistribution: "... development workflows and is, therefore, the ..."
- Delete "use FQL to". It's the CLI that users use to do these things, not FQL.
- Might emphasize the recent change by saying "is therefore **now** the recommended way".